client/GameForm.js

```diff
@@ -9,12 +9,13 @@ import TextField from '@material-ui/core/TextField/TextField';
 import { useInput } from './hooks/InputHook';
 
 const GameForm = ({ history }) => {
-    const {value: listType, bind: bindListType} = useInput('collection');
-    const {value: listOption, bind: bindListOption} = useInput('');
-    const {value: numPlayers, bind: bindNumPlayers} = useInput('');
-    const {value: minDuration, bind: bindMinDuration} = useInput('');
-    const {value: maxDuration, bind: bindMaxDuration} = useInput('');
-   
+    const { value: listType, bind: bindListType } = useInput('collection');
+    const { value: listOption, bind: bindListOption } = useInput('');
+    const { value: numPlayers, bind: bindNumPlayers } = useInput('');
+    const { value: minDuration, bind: bindMinDuration } = useInput('');
+    const { value: maxDuration, bind: bindMaxDuration } = useInput('');
+    const durationInputProps = { min: '0', max: '1440', step: '5' }; 

```

Comment:
It might be worth making these input values constants in their own right so we know what these magic numbers mean?